libraries/ts/src/sbv2.ts

```diff
@@ -2262,36 +2290,58 @@ export class OracleQueueAccount {
     return new OracleQueueAccount({ program, keypair: oracleQueueAccount });
   }
 
-  async setRewards(
-    params: OracleQueueSetRewardsParams
-  ): Promise<TransactionSignature> {
+  async setConfigTxn(
+    params: OracleQueueSetConfigParams & { authority?: Keypair }
+  ): Promise<anchor.web3.Transaction> {
+    const program = this.program;
     const authority =
       params.authority ?? this.keypair ?? programWallet(this.program);
-    return this.program.methods
-      .oracleQueueSetRewards({
-        rewards: params.rewards,
+    return program.methods
+      .oracleQueueSetConfig({
+        name: findKeyInObject(params, "name"),
```

Comment:
why not just params.name

Reply:
@ahermida asked a similar question: https://github.com/switchboard-xyz/switchboard-v2/pull/61#discussion_r977854883

My understanding is that `program.methods.oracleQueueSetConfig` expects all of these parameters to be set (`null` if we want the program to not update this parameter). So thats why I did this - as a consistent method to access the parameter (optionally transform) and otherwise return `null`. Basically it just didn't feel beautiful to have
`Number.isFinite(params.batchSize) ? params.batchSize : null` on every line.

Reply:
Resolved.